### ISSUE TITLE
OF support for Nvidia TX1 (ARM - amd64 - aarch64)

### ISSUE THREAD

**Issue by kaisark** [openframeworks/openFrameworks#5781]
Will there be OF support for installing on Nvidia TX1 (ARM - amd64 - aarch64)?

Should we go ahead and clone the arm7 config and port to TX1?

Thanks.

**Comment by arturoc** [comment]
yes please if you know what changes need to be done to make it work there please open a PR with them and we'll merge them
